### DISCUSSION TITLE
homebrew-cask-fonts for "All Users"

### DISCUSSION THREAD

**Discussion by jbaeck** [Homebrew/discussions#4138] [Getting started] [answered]
### Output of `brew config`

```shell
-
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

Hi, is there a possibility to install fonts (cask) with homebrew that in a multi-user environment all fonts installed (e.g. by an admin account) are automatically available to all users ?

**Comment by aavanian** [comment, marked as answer]
Old question but I had the same issue today and found a solution.

`brew install your-font --fontdir=/Library/Fonts/` works on OS X.
The man page also references it. I was looking to make it the default so putting this (without the `--appdir` argument) in your shell profile/config.

>HOMEBREW_CASK_OPTS
>
>    Append these options to all cask commands. All --*dir options, --language, --require-sha, --no-quarantine and --no-binaries are supported. For example, you might add something like the following to your ~/.profile, ~/.bash_profile, or ~/.zshenv:
>
>    export HOMEBREW_CASK_OPTS="--appdir=~/Applications --fontdir=/Library/Fonts"
```

**Reply by hitesh-pathak** [reply]
Thanks that helped me out :D
